tests/unit/serializers/application-test.js

```diff
@@ -17,6 +17,37 @@ module('application:pet', 'Unit | Serializer | application', function(hooks) {
     let snapshot = this.createSnapshot(model);
 
 
+    let json = {};
+    let relationship;
+    snapshot.eachRelationship((name, rel) => {
+      if (name === 'pets') {
+        relationship = rel;
+      }
+    });
+
+    serializer.serializeHasMany(snapshot, json, relationship);
+
+    assert.deepEqual(json, {
+      'pets': [
+        '3',
+        '10'
+      ]
+    }, 'We can call serializer.serializeHasMant directly');
+  });
+
+
+  test('Who needs the store and service APIs anyways... the snapshot ones are useful though', function(assert) {
+    let store = this.owner.lookup('service:store');
+    let serializer = this.owner.factoryFor('serializer:application').create({ store });
```

Comment:
are you sure you need to pass in `store`? isn't that injected automatically?

Reply:
In my own tests I was observing that serializer.store was not configured properly. I'll drop in a test case to demonstrate.